CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0 (2020-12-17)
+
+This is a major release concluding several release candidates and it delivers
+several major features to GA:
+* Add support for distributed hypertables
+* Add support for user-defined actions
+* Move enterprise features to community
+* Update Timescale License
+* Improve continuous aggregate API
+* Improve information views
+
+Since the last release candidate 4, it includes several minor improvements:
+* Properly block more unsupported functionalities on multinode
```

Comment:
Unclear what this refers to. What additional functionality are we blocking since RC4?

Reply:
* #2705 Block tableoid access on distributed hypertable
* #2730 Do not allow unique index on compressed hypertables